### PR TITLE
add: the example space model

### PR DESCRIPTION
**Pull request by mariobodemann**
Adds the example app space for an upcoming release.

Do not github release yet, since `the node example app` needs to be released first. This is for checking if the author fat fingered something.

### CODE REVIEW

File: the-example-app/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+What is `The Example App model`?
+================================
+
+The example app content model containts the model for all the example apps, demonstrating in a broader fashion some examples on how to use
+Contentful. If you want to see the model in action, we do have a hosted version here: https://contentful-example-app.herokuapp.com/
```
Comment (Roblinde) [inline the-example-app/README.md]: remove filler "do":  If you want to see the model in action, we have a hosted version here: https://contentful-example-app.herokuapp.com/

File: the-example-app/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Through our [command line app](https://github.com/contentful/contentful-cli) you can push this model to your own space. Doing so will be
```
Comment (Roblinde) [inline the-example-app/README.md]: I think removing the entire "Doing so..." statement actually makes it more readable:

Through our [command line app](https://github.com/contentful/contentful-cli) you can push this model to your own space:

```
 contentful space seed the-example-app
 ```

File: the-example-app/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+accomplished by this call to the command line app:
+
+```
+contentful space seed the-example-app
+```
+
+The app will then walk you through the needed steps in order to create a space based on it.
```
Comment (Roblinde) [inline the-example-app/README.md]: Remove filler "then" and simplify: 

The app will walk you through the steps needed in order to create a space.

File: the-example-app/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+Other apps
+==========
+
+The Node example app is not the only one. Soon™ there will be more available, all using the
```
Comment (Roblinde) [inline the-example-app/README.md:20]: "Soon™" love it 😄

**Review by Roblinde — APPROVED** [review]

File: the-example-app/README.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+What is `The Example App model`?
+================================
+
+The example app content model containts the model for all the example apps, demonstrating in a broader fashion some examples on how to use
+Contentful. If you want to see the model in action, we have a hosted version here: https://contentful-example-app.herokuapp.com/
```
Comment (floelhoeffel) [inline the-example-app/README.md]: update the URL

**Review by floelhoeffel — APPROVED** [review]
Readme is fine with me - I will add a few things once in GH.

**Pull request merged**